vendor/src/github.com/docker/distribution/registry/client/repository.go

```diff
@@ -604,6 +621,45 @@ func (bs *blobs) Resume(ctx context.Context, id string) (distribution.BlobWriter
 	panic("not implemented")
 }
 
+func (bs *blobs) Mount(ctx context.Context, sourceRepo string, dgst digest.Digest) (distribution.Descriptor, error) {
```

Comment:
This is currently commented in `vendor/src/github.com/docker/distribution/blobs.go`, but that can be elaborated on if necessary